dobson/utils.py

```diff
@@ -35,13 +35,19 @@
     exit(1)
 
 
-def get_devices():
-    """Get all recognized connected devices"""
+def get_devices(require_presence=True):
```

Comment:
I'd name this something like `return_all` or something, and have the default be `False`, then you can just do something like `if found_device.presence or return_all` and it's easier to read and reason about. I feel like the name could be different though, `return_all` is probably not the best name.

Reply:
Good idea, I think I'll change it to be `return_all`, that seems to make semantic sense.